hopglass-server.js

```diff
@@ -497,6 +498,8 @@ function getMetrics(stream) {
       counter_clients += get(n, 'statistics.clients.total')
     }
 
+    nodeNames[_.get(n, ['nodeinfo', 'node_id'])] = _.get(n, ['nodeinfo', 'hostname'])
```

Comment:
The iterated node's id is stored in 'k'. No need to get it.
Also the default should be undefined. 
I suggest '`nodeNames[k] = _.get(n, ['nodeinfo', 'hostname'], undefined)`'.